app/lib/actions/katello/upstream_subscriptions/bind_entitlements.rb

```diff
@@ -0,0 +1,28 @@
+module Actions
+  module Katello
+    module UpstreamSubscriptions
+      class BindEntitlements < Candlepin::Abstract
+        def plan(organization_id = nil, pools = [])
+          sequence do
+            plan_self(organization_id: organization_id, pools: pools)
+            return unless pools.any?
```

Comment:
I decided to fail() entirely if entitlement_ids were empty in my code. We should probably do the same here - also addresses justin's comment about the placement of this return which would still run the plan_self unnecessarily

Reply:
(copied from above)

> I was thinking that users would be able to see what they did wrong in the task output rather than just not creating the task at all. Would the task still be created if we didn't call plan_self?

Reply:
Sorry, missed that! It looks like currently it shows success even though we didn't actually bind anything. To fail or have a successful noop? 🤔